client/keys/import.go

```diff
@@ -38,3 +40,22 @@ func ImportKeyCommand() *cobra.Command {
 		},
 	}
 }
+
+func ImportKeyHexCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "import-hex <name> <hex>",
+		Short: "Import private keys into the local keybase",
+		Long:  "Import hex encoded private key into the local keybase.\nSupported key-types can be obtained with:\n<appd> list-key-types",
```

Comment:
small nit: Instead of `<appd>` use `version.AppName`. Like here: https://github.com/cosmos/cosmos-sdk/blob/0e8cef5795cefadcea83a0b4815c843dc70ecaa9/x/bank/client/cli/tx.go#L95

Reply:
@JulianToledano could you implement this? Then we can merge :)